packages/monaco/src/typings/monaco/index.d.ts

```diff
@@ -743,8 +743,125 @@ declare module monaco.quickOpen {
         setShowBorder(showBorder: boolean): void;
         getEntry(): QuickOpenEntry | undefined;
     }
+
+    export interface INextIterator<T> {
```

Comment:
please expose only what is used and remove everything what is not used in this PR

Reply:
Removed